index.html

```diff
@@ -0,0 +1,192 @@
+                </div>
+            </div>
+            <div class="col col-md-2">
+                <div class="row align-items-center box-container py-1 px-2 item-container gy-md-2">
+                    <div class="col-2 col-md-12 text-md-center"><i class="fa-solid fa-house-laptop icons-color"></i></div>
+                    <div class="col-3 col-md-12 text-md-center "><span class="fw-bold semi-title">Work</span></div>
+                    <div class="col-7 col-md-12 ps-3 px-md-0 pe-0"><p class="fw-bold text-white text-center">Try creating your own work using open source 
+                        license rather than just watching.
+                    </p>
+                    </div>
+                </div>
+            </div>
+            <div class="col col-md-2">
+                <div class="row align-items-center box-container py-1 px-2 item-container gy-md-2">
+                    <div class="col-2 col-md-12 text-md-center"><i class="fa-solid fa-volume-high  icons-color"></i></div>
+                    <div class="col-3 col-md-12 text-md-center "><span class="fw-bold semi-title">Lecture</span></div>
+                    <div class="col-7  col-md-12 ps-3 px-md-0 pe-0"><p class="fw-bold text-white text-center">Listen to the speakers from various countries
+                        about the messages of sharing and opening.</p>
+                    </div>
+                </div>
+            </div>
+            <div class="col col-md-2">
+                <div class="row align-items-center box-container py-1 px-2 item-container gy-md-2">
+                    <div class="col-2 col-md-12 text-md-center"><i class="fa-solid fa-circle-nodes icons-color"></i></div>
+                    <div class="col-3 col-md-12 text-md-center "><span class="fw-bold semi-title">Network</span></div>
+                    <div class="col-7 col-md-12 ps-3 pe-0"><p class="fw-bold text-white text-center">Get opportunities to network with CC affilliates around the 
+                        world.
+                    </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div id="join-button" class=" text-md-center d-md-none">
+            <button type="button">Join Tech Innovation</button>
+        </div>
+        <div class="see-program-text-container d-none d-md-block">
+            <a href="" class=""><h5 class="text-white text-uppercase">See the Whole Program</h5></a>
+        </div>
+    </section>
+
+    <!-- FEATURE SPEAKER SECTION  -->
+    <section id="feature-speaker-section" class="">
+        <h2 class="text-center text-black fw-bold " id="feature-title">Featured Speakers</h2>
+        <div class="line"></div>
+        <div class="row speaker-container row-cols-1 row-cols-md-2 mt-5" id="speaker-container">
+ 
+        </div>
+    </section>
+
+    <!-- SPONSOR SECTION  -->
+    <section class="container-fluid" id="sponsor-section">
+        <h2 class="text-center " id="sponsor-title">Partner And Sponsor</h2>
```

Comment:
- [ ] Please note that your divider is much longer than in the Figma. Your mandate is to customize the content but reatin the design.
![partner](https://github.com/HtetWaiYan7191/capstone-project/assets/99656687/1ebb6f69-caf6-4ce2-ae34-a1561ebf2e5d)